pkgs/servers/klipper/klipper-firmware.nix

```diff
@@ -7,7 +7,7 @@
 , libusb1
 , wxGTK32
 , python3
-, gcc-arm-embedded
+, gcc-arm-embedded-11
```

Comment:
Wouldn't it be better to override this in `all-packages.nix` like this?

```diff
diff --git a/pkgs/top-level/all-packages.nix b/pkgs/top-level/all-packages.nix
index eacb266e2ff7..f039b169a82c 100644
--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -5783,7 +5783,7 @@ with pkgs;
 
   klipper = callPackage ../servers/klipper { };
 
-  klipper-firmware = callPackage ../servers/klipper/klipper-firmware.nix { };
+  klipper-firmware = callPackage ../servers/klipper/klipper-firmware.nix { gcc-arm-embedded = gcc-arm-embedded-11; };
 
   klipper-flash = callPackage ../servers/klipper/klipper-flash.nix { };
```

Reply:
Yeah makes more sense. Updated it

Reply:
@vtuan10 I don't have too much experience with packaging stuff in nixpkgs. Both your method and mine fix this particular issue, but I don't see myself as an authority to say which one is better in any way, or why.

Reply:
Your way is better IMO as this is a current workaround until klipper fixes this issue, no need to change the `klipper-firmware` derivation 👍